aws-rds-dbparametergroup/src/test/java/software/amazon/rds/dbparametergroup/AbstractTestBase.java

```diff
@@ -49,6 +49,32 @@ public class AbstractTestBase {
     protected static final Map<String, Object> PARAMS;
     protected static final Map<String, Object> RESET_PARAMS;
     protected static final RequestLogger EMPTY_REQUEST_LOGGER;
+    protected static final List<Parameter> MANY_CURRENT_PARAMETERS_SORTED;
+    protected static final List<Parameter> MANY_DEFAULT_PARAMETERS_SORTED;
```

Comment:
Let's keep these off the abstract test base if these are only needed for 1 test. I believe the test method name might reflect that we are testing against a massive collection of parameters.